app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/welcome/WelcomeScreen.kt

```diff
@@ -121,15 +119,15 @@ private fun WelcomeScreenContent(
         Spacer(modifier = Modifier.weight(1f))
 
         HorizontalPager(state = pagerState) { index ->
-            if (isLandscape) {
-                WelcomeCardLandscape(
+            if (LocalConfiguration.current.isPortrait) {
+                WelcomeCardPortrait(
                     state = state.pages[index],
-                    modifier = Modifier.padding(horizontal = horizontalPadding),
+                    modifier = Modifier.standardHorizontalMargin(landscape = 128.dp),
```

Comment:
This shouldn't need an override for landscape, right?

Reply:
Technically correct, the best kind of correct!